javatests/build/bazel/tests/integration/BazelBaseTestCaseTest.java

```diff
@@ -72,6 +74,14 @@ public void scratchFileShouldCreateFileAndWorkspaceContentsContainThatFile() thr
     org.hamcrest.MatcherAssert.assertThat(actualScratchFileContent, is(optionalWithValue(equalTo(content))));
   }
 
+  @Test
+  public void getRunfileReturnTheFile() {
+      File runfile = getRunfile("build_bazel_integration_testing", "tools", "BUILD");
+
+      boolean actualIsFileExists = runfile.exists();
```

Comment:
do we really need this intermediate variable?

Reply:
inlined